composer.json

```diff
@@ -5,17 +5,18 @@
     "homepage": "https://github.com/studiopress/genesis-custom-blocks",
     "license": "GPL-2.0",
     "require": {
-        "php": "^5.6 || ^7"
+        "php": "^5.6 || ^7 || ^8"
     },
     "require-dev": {
-        "brain/monkey": "2.4.0",
-        "dealerdirect/phpcodesniffer-composer-installer": "0.7.0",
-        "mikey179/vfsstream": "1.6.8",
-        "mockery/mockery": "1.3.1",
-        "phpcompatibility/phpcompatibility-wp": "2.1.0",
-        "squizlabs/php_codesniffer": "3.5.3",
-        "wp-coding-standards/wpcs": "2.2.0",
-        "phpunit/phpunit": "7.5.2"
+        "brain/monkey": "^2",
```

Comment:
Uses a version of `^2` instead of `2.4.0`, as it's fine for composer to get the latest `2.*`